doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -455,3 +456,11 @@ which attributes to include in the response to the query) and unary attribute fi
 `expression` (to return only entities which have these attributes). Use them instead.
 
 [Top](#top)
+
+## Additional fields in `httpInfo` for custom notifications
+
+* includePayload: Additional field in `httpInfo` for custom notifications to check not including payload or 0-length requests.
+set 'true' in the case of not including payload in the request.
```

Comment:
```suggestion
Orion supports a boolean field `includePayload` in `httpInfo` for custom notifications that can be used to avoid payload in the outgoing notification. If `includePayload` is set to `true` then no payload is included in the notification (and `payload` field, if used, is ignored).

Default value if `includePayload` is omitted is `false`.
```

Reply:
Updated